dashboard/test/ui/features/projects.feature

```diff
@@ -234,3 +231,19 @@ Scenario: Starwars Flow
   And I wait to see "#footerDiv"
   Then I should see title "Code Ninja III: Revenge of the Semicolon - Play Lab"
   And element "#codeWorkspace" is hidden
+
+@eyes
+@as_student
+Scenario: My Projects
+  When I open my eyes to test "My Projects page"
+  And I am on "http://studio.code.org/projects/"
```

Comment:
this is out of experiment mode?

Reply:
yep! just recently, in https://github.com/code-dot-org/code-dot-org/pull/19242